packages/components/src/components/domainproperties/models.ts

```diff
@@ -311,6 +311,8 @@ export const DEFAULT_DOMAIN_FORM_DISPLAY_OPTIONS = {
     hideAddFieldsButton: false,
     disableMvEnabled: false,
     hideImportData: false,
+    hideDeleteIcon: false,
+    lockName: false,
```

Comment:
To be more consistent with the other property names in this model, I'd probably change this to `disableNameInput`